src/appellate/utils.js

```diff
@@ -272,14 +274,14 @@ let APPELLATE = {
       // store extra information on anchors to use it while handling the onClick listener
       let docId = PACER.getDocumentIdFromUrl(clonedNode.href);
 
-      clonedNode.setAttribute('data-pacer_doc_id', docId);
+      clonedNode.dataset.pacerDocId = docId;
       if (doDoc && doDoc.doc_id) {
-        clonedNode.setAttribute('data-pacer_dls_id', doDoc.doc_id);
+        clonedNode.dataset.pacerDlsId = doDoc.doc_id;
       }
-      clonedNode.setAttribute('data-pacer_case_id', pacerCaseId);
-      clonedNode.setAttribute('data-pacer_tab_id', tabId);
-      clonedNode.setAttribute('data-document_number', docNum ? docNum : docId);
-      clonedNode.setAttribute('data-attachment_number', attNumber);
+      clonedNode.dataset.pacerCaseId = pacerCaseId;
+      clonedNode.dataset.pacerTabId = tabId;
+      clonedNode.dataset.documentNumber = docNum ? docNum : docId;
+      clonedNode.dataset.attachmentNumber = attNumber;
```

Comment:
Is the changing of snake_case to camelCase OK here? Looks scary, but I imagine you wouldn't do this unless it was equivalent?

Reply:
Yeah, it's how the dataset API is defined, cf. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#name_conversion
I guess it's accomplished with a snazzy DOMStringMap thing. I don't disagree that it is potentially confusing, yet it still seemed better to use the API for writing this data rather than hand-rolling it. Shrug.

Reply:
I get it. I guess HTML is better with dashes and JS is better with camels. Great. I'll merge.